### DISCUSSION TITLE
Table multiple sort

### DISCUSSION THREAD

**Discussion by Dwuuu** [grafana/grafana#57502] [Dashboards, Panels & Visualizations]
I've created a new table, and I want to multisort it by its value. Columns such as A ascending order, B descending order, C ascending order

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
